docs/guides/location-service/fragments/setting-up-your-app-android.md

```diff
@@ -0,0 +1,210 @@
+1. Install the [Amplify CLI](https://docs.amplify.aws/cli/start/install) by running the following command:
+
+    ```bash
+    npm install -g @aws-amplify/cli
+    ```
+
+1. Initialize a new Amplify project by running the following command from the root directory of your app:
+
+    ```bash
+    amplify init
+    ```
+
+1. Create a Cognito Identity Pool. It will be used to authenticate your app users and authorize their access to Amazon Location Service. To start provisioning authentication resources in the backend, go to your project directory and run the following command:
+
+    ```bash
+    amplify add auth
+    ```
+
+1. When prompted, provide the following information:
+
+    ```console
+    ? Do you want to use the default authentication and security configuration? 
+      `Default configuration`
+    ? How do you want users to be able to sign in? 
+      `Username`
+    ? Do you want to configure advanced settings? 
+      `No, I am done.`
+    ```
+
+1. Run the following command to push your changes to the cloud. When completed, the awsconfiguration.json file will be updated to reference your newly provisioned backend auth resources.
+
+    ```bash
+    amplify push
+    ```
+
+## Initializing the SDK
+
+To use your new Amazon Cognito identity pool, add the following code to initialize AWSMobileClient in your Activity's onCreate method:
+
+<amplify-block-switcher>
+
+<amplify-block name="Java">
+
+```java
+AWSMobileClient.getInstance().initialize(getApplicationContext(), new Callback<UserStateDetails>() {
+    @Override
+    public void onResult(UserStateDetails userStateDetails) {
+        Log.i("QuickStart", "onResult: " + userStateDetails.getUserState());
+    }
+
+    @Override
+    public void onError(Exception e) {
+        Log.e("QuickStart", "Initialization error: ", e);
+    }
+});
+```
+
+</amplify-block>
+<amplify-block name="Kotlin">
+
+```kotlin
+AWSMobileClient.getInstance().initialize(applicationContext, object : Callback<UserStateDetails> {
+    override fun onResult(userStateDetails: UserStateDetails) {
+        Log.i("QuickStart", "onResult: " + userStateDetails.userState)
+    }
+
+    override fun onError(e: Exception) {
+        Log.e("QuickStart", "Initialization error: ", e)
+    }
+})
+```
+
+</amplify-block>
+</amplify-block-switcher>
+
+This allows guest users access to your backend resources. 
+
+You have now successfully setup and configured your app to use Amazon Location Service!
+
+## Using the Amazon Location Service APIs
+
+In order to access Amazon Location Service APIs you will need to create resources. You can create resources using the [Amazon Location Service console](http://console.aws.amazon.com/location/home) or using the [AWS Command Line Interface (CLI)](https://aws.amazon.com/cli/). Once you have created the resources, you can use these resources by calling the [APIs](https://aws-amplify.github.io/aws-sdk-android/docs/reference/). 
```

Comment:
We haven't updated our javadoc (https://aws-amplify.github.io/aws-sdk-android/docs/reference/) since 2.16.12, so it doesn't have any Location API documentation.  I think we either need to remove this link, or get the javadoc up to date (I don't know what the process is for that, yet)

Reply:
@richardmcclellan resolved this 👍 .